requirements.txt

```diff
@@ -1,23 +1,19 @@
-Django==1.11.29
+Django==3.2
 Markdown==2.6.8
-Pillow==4.1.1
-pyyaml==4.2b1
+Pillow==8.3.2
```

Comment:
Bump to `9.0.1` to resolve the outstanding dependabot alert.

Reply:
Done! builds and runs fine.